csrc/quantization/cutlass_w8a8/grouped_mm_c3x.cuh

```diff
@@ -0,0 +1,153 @@
+#pragma once
+
+#include "cutlass/cutlass.h"
+
+#include "cutlass/gemm/collective/collective_builder.hpp"
+#include "cutlass/epilogue/collective/collective_builder.hpp"
+#include "cutlass/gemm/device/gemm_universal_adapter.h"
+
+#include "cutlass_extensions/epilogue/scaled_mm_epilogues_c3x.hpp"
+#include "cutlass_extensions/common.hpp"
+#include "get_group_starts.cuh"
+
+using namespace cute;
+
+#if defined __CUDA_ARCH__ && __CUDA_ARCH__ >= 900
```

Comment:
This needs to be:
```
#if defined __CUDA_ARCH__ && __CUDA_ARCH__ == 900
```
Otherwise it will break on Blackwell, as these H100 CUTLASS kernel are not forward-compatible